SkillChat.Client.ViewModel/MainWindowViewModel.cs

```diff
@@ -469,7 +509,43 @@ public MainWindowViewModel()
             ProfileViewModel.SignOutCommand = SignOutCommand;
             ProfileViewModel.LoadMessageHistoryCommand = LoadMessageHistoryCommand;
         }
+        /// <summary>
+        /// Метот выхода из режима редактирования
+        /// </summary>
+        private void EndEditMethod()
+        {
+            if (MessageText != null && idEditMessage != null)
```

Comment:
А что это за ситуация когда MessageText равен null? Зачем это сдесь проверяем?
```suggestion
            if (idEditMessage != null)
```